src/main/java/it/gov/pagopa/arc/service/bizevents/BizEventsServiceImpl.java

```diff
@@ -63,4 +77,31 @@ public Resource retrieveTransactionReceiptFromBizEvents(String transactionId) {
         String retrievedUserFiscalCode = SecurityUtils.getUserFiscalCode();
         return bizEventsConnector.getTransactionReceipt(retrievedUserFiscalCode, transactionId);
     }
+
+    @Override
+    public NoticesListResponseDTO retrievePaidListFromBizEvents(String continuationToken, Integer size, Boolean isPayer, Boolean isDebtor, String orderBy, String ordering) {
+        NoticesListDTO noticesListDTO = NoticesListDTO.builder().build();
+        NoticesListResponseDTO noticesListResponseDTO =  NoticesListResponseDTO.builder().noticesListDTO(noticesListDTO).build();
+        String retrievedContinuationToken = null;
+
+        String retrievedUserFiscalCode = SecurityUtils.getUserFiscalCode();
```

Comment:
This is a contextual information retrieved by the HTTP source...
It could be better to retrieve it from the Controller, and passing to the service as a parameter

Reply:
I fetch it from the BizEvents connector and map it to a BizEventsPaidResponseDTO, then I can fetch it and map it to the NoticeListResponseDTO to return it into our client, there is no direct connection between our controller and the BizEvents connector other than through the service